__tests__/unit/heaps/Heap.es6

```diff
@@ -0,0 +1,118 @@
+jest.autoMockOff();
+let IM = require('immutable');
+let Heap = require('../../../src/heaps/Heap');
+
+describe("Heap Operations", ()=>{
```

Comment:
The style guide isn't clear on es6 conventions, but I propose that we follow the same principles as we do with es3-5. 

I suggest that we leave extra whitespace for our functions, e.g., `() => {}`.